src/current/v23.1/alter-backup-schedule.md

```diff
@@ -50,13 +50,17 @@ You can use the backup options in this table to control the behavior of your bac
 
 You can use the schedule options in this table to control the behavior of your backup schedule. See [Apply different options to scheduled backups](#apply-different-options-to-scheduled-backups) for an example.
 
-{% include {{ page.version.version }}/backups/schedule-options.md %}
+Option                     | Value                                   | Description
```

Comment:
The diff doesn't make this totally obvious, but I removed the `first_run` and `ignore_existing_backups` options here.

Reply:
the 2 options are correctly removed.